modules/apps/journal/journal-web/src/main/resources/META-INF/social/liferay-social.xml

```diff
@@ -2,6 +2,12 @@
 <!DOCTYPE liferay-social PUBLIC "-//Liferay//DTD Social 7.4.0//EN" "http://www.liferay.com/dtd/liferay-social_7_4_0.dtd">
 
 <liferay-social>
+	<activity>
+		<model-name>com.liferay.journal.model.JournalArticle</model-name>
```

Comment:
@thuycao-gs 
Please refer to this file https://github.com/hongvo2308/liferay-portal/blob/master/modules/apps/blogs/blogs-web/src/main/resources/META-INF/social/liferay-social.xml and update your implementation.
Thanks